docs/operating-eck/licensing.asciidoc

```diff
@@ -9,6 +9,8 @@ endif::[]
 
 When you install the default distribution of ECK, you receive a Basic license. Any Elastic stack application you manage through ECK will also be Basic licensed. Go to https://www.elastic.co/subscriptions to see which features are included in the Basic license for free.
 
+IMPORTANT: ECK is offered in two licensing tiers only: Basic and Enterprise. Similar to the Elastic Stack, customers can download and use ECK for free, but a subscription is required for support. ECK support requires an Enterprise subscription.
```

Comment:
Some quick modifications -
"IMPORTANT: ECK is only offered in two licensing tiers: Basic and Enterprise. Similar to the Elastic Stack, customers are able to download and use ECK with a basic license for free. A paid enterprise subscription is required for support."

Reply:
```suggestion
"IMPORTANT: ECK is only offered in two licensing tiers: Basic and Enterprise. Similar to the Elastic Stack, customers are able to download and use ECK with a basic license for free. Basic license users can avail of support from Github or through our community at https://discuss.elastic.co. A paid enterprise subscription is required to engage the elastic support team. "
```